test/FunctionsV2/WebJobs.Extensions.DurableTask.Tests.V2.csproj

```diff
@@ -27,11 +27,21 @@
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
     <PackageReference Include="StyleCop.Analyzers" Version="1.1.*" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.Emulator" Version="2.1.3" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.Redis" Version="0.1.1-alpha" />
   </ItemGroup>
 
   <ItemGroup>
     <Compile Include="..\Common\*.cs" />
   </ItemGroup>
+
+  <PropertyGroup Condition="true">
+    <DefineConstants>NET_CORE</DefineConstants>
```

Comment:
I'm thinking of renaming this to FUNCTIONS_V2, so it is more explicit.

Reply:
Probably wouldn't be a bad idea to do something similar for conditional compilation in the source code as well.